index.html

```diff
@@ -289,10 +290,11 @@ <h2 class="contact-title-LG">Let’s chat!</h2>
                     If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
                 </p>
             </div>
-            <form id="form1" action="https://formspree.io/f/xbjwvdqb" method="post">
+            <form id="contact-form" action="https://formspree.io/f/xbjwvdqb" method="post">
                 <input type="text" id="name" name="name" maxlength="30" placeholder="Text input" required />
                 <br />
                 <br />
+                <p id="errorDisplay"></p>
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required />
                 <br />
                 <br />
```

Comment:
- [ Optional] Your portfolio looks awesome, it will look even better if you can add a margin-left inside your form inputs.